src/integration/common/header.js

```diff
@@ -12,10 +12,20 @@ export default () => {
       const navigationItemEls = document.querySelector(
         'header nav ul > li > a',
       );
-
       expect(navigationItemEls).toBeInTheDocument();
     });
 
+    const navigationLinks = document.querySelectorAll('header nav a');
+    navigationLinks.forEach(navigationLink => {
+      it('I can see a navigation link', () => {
```

Comment:
This might make it easier to spot which link is broken when running this test
```suggestion
      it(`I can see navigation link ${navigationLink.textContent}`, () => {
```

Reply:
I discovered that if you pass a string as the first param to `toMatchSnapshot()`, it puts that text in the snapshot description - e.g. https://github.com/bbc/simorgh/blob/680b1ef677529efe4828b3d3593d8af0df5ad533/src/integration/pages/articles/mundo/__snapshots__/amp.test.js.snap

However, your suggestion is much cleaner, so I have updated both occurrences - thanks!